src/Compilers/CSharp/Portable/BoundTree/NullabilityRewriter.cs

```diff
@@ -8,5 +13,68 @@ protected override BoundExpression VisitExpressionWithoutStackGuard(BoundExpress
         {
             return (BoundExpression)Visit(node);
         }
+
+        public override BoundNode VisitBinaryOperator(BoundBinaryOperator node)
+        {
+            return VisitBinaryOperatorBase(node, (binary, left, right, type) => node.Update(node.OperatorKind, node.ConstantValueOpt, node.MethodOpt, node.ResultKind, left, right, type));
+        }
+
+        public override BoundNode VisitUserDefinedConditionalLogicalOperator(BoundUserDefinedConditionalLogicalOperator node)
+        {
+            return VisitBinaryOperatorBase(node, (binary, left, right, type) => node.Update(node.OperatorKind, node.LogicalOperator, node.TrueOperator, node.FalseOperator, node.ResultKind, left, right, type));
```

Comment:
>(binary, left, right, type) => node.Update(node.OperatorKind, node.LogicalOperator, node.TrueOperator, node.FalseOperator, node.ResultKind, left, right, type) [](start = 49, length = 158)

It looks like this lambda has all the same problems as the one in the function above.